.github/workflows/reusable-ci-jobs.yml

```diff
@@ -128,6 +136,10 @@ jobs:
       - name: checkout
         uses: actions/checkout@v4
 
+      - name: Change permissions
+        run: |
+          sudo chmod -R 777 /opt/hostedtoolcache
```

Comment:
What is the reason for this?

Reply:
![image](https://github.com/tari-project/tari-launchpad/assets/42658387/16bf74c0-4f0b-42b4-b863-306f24464170)